test/e2e/config/e2e_conf.yaml

```diff
@@ -0,0 +1,95 @@
+---
+images:
+- name: ghcr.io/weaveworks-liquidmetal/cluster-api-provider-microvm:e2e
+  loadBehavior: mustLoad
+- name: quay.io/jetstack/cert-manager-cainjector:v1.7.2
+  loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-webhook:v1.7.2
+  loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-controller:v1.7.2
+  loadBehavior: tryLoad
+- name: registry.k8s.io/cluster-api/cluster-api-controller:v1.1.5
+  loadBehavior: tryLoad
+- name: registry.k8s.io/cluster-api/kubeadm-bootstrap-controller:v1.1.5
+  loadBehavior: tryLoad
+- name: registry.k8s.io/cluster-api/kubeadm-control-plane-controller:v1.1.5
+  loadBehavior: tryLoad
```

Comment:
So I found that the framework will load these images into the kind cluster, but wont pull them down. So you have to just have them on the machine first which is a pain.
IDK if the capa or oci providers use some extra framework trick, I have not found it yet. For now I am going to script something i guess, or write a helper func in the tests.

Reply:
eh mine is better ;)